libpod/oci_linux.go

```diff
@@ -417,6 +427,14 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string, res
 		}
 		logrus.Debugf("Received container pid: %d", ss.si.Pid)
 		if ss.si.Pid == -1 {
+			data, err := ioutil.ReadFile(ociLog)
+			if err == nil {
+				var ociErr ociError
+				if err := json.Unmarshal(data, &ociErr); err == nil {
+					return errors.Wrapf(ErrInternal, "container create failed: %s", strings.Trim(ociErr.Msg, "\n"))
+				}
+			}
+			// If we failed to parse the JSON errors, then print the output as it is
 			if ss.si.Message != "" {
 				return errors.Wrapf(ErrInternal, "container create failed: %s", ss.si.Message)
```

Comment:
Here I also just return 
return errors.Errorf(ss.si.Message)
This is not an internal error and we did create the container,

Reply:
fixed